applications/visit/app.json

```
@@ -0,0 +1,93 @@
+    "owner": "${apiUserId}",
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/visit/visit.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "frontera",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
+        "execSystemLogicalQueue": "small",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "mpiCmd": null,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [],
+            "containerArgs": [],
+            "schedulerOptions": [],
+            "envVariables": [
+                {
+                    "key": "desktop_resolution",
+                    "value": "1280x800",
+                    "description": "Set the desktop screen size for your visualization session.",
+                    "inputMode": "REQUIRED",
+                    "notes": {
+                        "label": "Desktop Resolution",
+                        "enum_values": [
+                            {
+                                "800x600": "800x600"
+                              },
+                              {
+                                "1280x800": "1280x800"
+                              },
+                              {
+                                "1920x1080": "1920x1080"
+                              },
+                              {
+                                "1920x1200": "1920x1200"
+                              },
+                              {
+                                "1600x1200": "1600x1200"
+                              },
+                              {
+                                "2560x1600": "2560x1600"
+                              }
+                        ]
+                    }
+                }
+            ]        
+        },
+        "fileInputs": [
+            {
+                "name": "Working Directory",
+                "description": "The directory containing the files that you want to work on. This directory and its files will be copied to where your interactive session runs. You can drag the link for the directory from the Data Browser on the left, or click the 'Select Input' button and then select the directory.",
```

Let's remove the text referencing drag and drop, that will be gone in v3
```suggestion
                "description": "The directory containing the files that you want to work on. This directory and its files will be copied to where your interactive session runs. Click the 'Select Input' button and then select the directory you wish to work from.",
```